.gitignore

```
@@ -4,3 +4,12 @@
 /gatewayApp/gateway/__pycache__
 gatewayApp/utils/__pycache__
 gatewayApp/utils/key.priv
+/Message/__pycache__
+nodeApp/node/__pycache__
+app/__pycache__/__init__.cpython-312.pyc
+app/__pycache__/__main__.cpython-312.pyc
+Message/__pycache__/Message.cpython-312.pyc
```

I think its better to do general:
```suggestion
Message/__pycache__/
```
similar elsewere